docs/data_managers/install/configuration.md

```diff
@@ -219,6 +240,20 @@ configuration elements:
 : When this element is `true`, all datasets will be published to the testing sandbox
   site. Set this to `false` when you are ready to actually start publishing datasets.
 
+### XML configuration
+
+The `safedata_zenodo` tool can be used to generate a geo-spatial XML metadata file for a
+dataset. This is relatively high-level metadata that just includes the temporal and
+spatial bounds of the data, along with some contact and access details.
+
+We recommend that this file is included when datasets are published. If you want to do
+this, you will need to update this section with the details for your own project. You
+will likely only need to update the `contact...`, `pointofcontact...`, `topicCategories`
+and `lineageStatement` elements. The `contact...` details provide a general contact for
```

Comment:
`ContactCountry`, `contactOrcID` and `citationMDIdentifier` also likely to change right? Or is it that they are not essential to include?

Reply:
I'm thinking we just get rid of all of those `contact...`, `pointofcontact...` ones and just use the details used to publish to Zenodo. That seems consistent and massively simplifies the XML section. I'm not even sure what the `citationMDIdentifier` is now - so that needs handling!